### ISSUE TITLE
Documentation Incomplete/Inaccurate

### ISSUE THREAD

**Issue by mrobertmiller** [MicrosoftDocs/azure-devops-docs#12906]
As I previously reported 2 months ago, the "Sync your fork" section of the documentation is still broken/incomplete/inaccurate.  It does not properly state how to sync your forked repo with the upstream, leaving the reader lost and hunting around on how to do it.  Do you plan on fixing this or not???

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: d5a579e1-f493-7315-75a6-62eefd9877ad
* Version Independent ID: 05a7e8ca-00f0-cd31-87c4-47d9795acb28
* Content: [Fork your repository - Azure Repos](https://learn.microsoft.com/en-us/azure/devops/repos/git/forks?view=azure-devops&tabs=visual-studio#clone-locally)
* Content Source: [docs/repos/git/forks.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/main/docs/repos/git/forks.md)
* Service: **azure-devops-repos**
* Sub-service: **azure-devops-repos-git**
* GitHub Login: @vijayma
* Microsoft Alias: **vijayma**

**Comment by WilliamAntonRohm** [comment]
Duplicate of https://github.com/MicrosoftDocs/azure-devops-docs/issues/12751.

cc @vijayma for awareness

**Comment by mrobertmiller** [comment]
You marked this as completed but made no corrections to the documentation, even though I have reported it as defective twice over a period of months that you have had to fix it.  When this was updated, someone botched the rewrite of the "Sync" section to match the current implementation of Visual Studio 2022.   What is written in that section is just a bunch of gibberish and does not even close to describing the simple step to sync a fork, requiring the reader to go hunting for how to do it.  It is incompetent of you to close this without fixing the documentation to match the tool.  But I at this point, I know how to sync, so I no longer care that you don't care to do your job.  Thanks for wasting more of my time.
